docs/content/preview/api/ysql/the-sql-language/statements/_index.md

```diff
@@ -70,6 +70,7 @@ The YSQL statements are compatible with the SQL dialect that PostgreSQL supports
 | [`DROP SEQUENCE`](ddl_drop_sequence) | Delete a sequence generator |
 | [`DROP SERVER`](ddl_drop_server) | Drop a foreign server |
 | [`DROP TABLE`](ddl_drop_table) | Delete a table from a database |
+| [`DROP INDEX`](ddl_drop_index) | Delete an index from a database |
```

Comment:
put in alphabetical order

Reply:
Fixed in https://github.com/yugabyte/yugabyte-db/pull/17543/commits/73e51a66d4675bea007c4c47f3d03e6ef363bbb5